Src/Base/AMReX_Version.cpp

```diff
@@ -0,0 +1,14 @@
+#include <AMReX.H>
```

Comment:
We don't need to include AMReX.H here.

Reply:
I would, because it declares `amrex::Version ();`, no?

Reply:
We could move that one to `AMReX_Version.H` though :)

Reply:
You can define functions without declaring them.

Reply:
If we move that to AMReX_Version.H, the users would have to include a new header to be able to call amrex::Version().

Reply:
Oh right, then let me remove my last commit again :D

Reply:
I would agree that we should include `AMReX_Version.H` so that the function declaration is include, if it were C.  But because of function overloading in C++, it's not really useful to have the declaration in AMReX_Version.cpp.

Reply:
> You can define functions without declaring them? Was that @WeiqunZhang 's comment?

Probably. I always include the declaration when I define things, but you can remove including `AMReX.H` if you think it's unnecessary here.

Reply:
Updated. I will stop push-ing to the branch now :)

Reply:
Ah dang. In that case let me rewrite the `.cpp` file once more:
```
../Src/Base/AMReX_Version.cpp:7:13: error: use of undeclared identifier 'amrex'
```

Reply:
@WeiqunZhang fixed. now for real